### ISSUE TITLE
make a new release based on current libraries / firmware

### ISSUE THREAD

**Issue by ThomasWaldmann** [ecocurious/MultiGeiger#469]
Got an email from a user with strange wifi problems which suggested that they might be fixed on newer ESP IDF.

So, guess it would be good to make a new release based on the current stuff.

**Comment by rexfue** [comment]
I guess, You should do it

**Comment by ThomasWaldmann** [comment]
@rexfue yeah, just need to install the current stuff on the macbook first.

**Comment by MMBytes** [comment]
Hi @ThomasWaldmann, any chance for a new release? Thanks

**Comment by ThomasWaldmann** [comment]
@MMBytes last time I tried I ran into strange build issues I couldn't resolve. 

I'll try again soon....

**Comment by MMBytes** [comment]
> @MMBytes last time I tried I ran into strange build issues I couldn't resolve.
> 
> I'll try again soon....

Thank you!

**Comment by MMBytes** [comment]
Hi @ThomasWaldmann, any luck with this? 
Many thanks

**Comment by eku** [comment]
As I understand it, no version for Espressif's SDK is specified in platformio.ini and therefore the latest version is automatically used. However, this is not compatible with the source code version.


There is nothing wrong with using an older version. But which one?

Pinning the versions in `platformui.ini` solves my compilation problem

```
platform = espressif32@6.7.0
....
  h2zero/NimBLE-Arduino@1.4.1
```

```bash
Linking .pio/build/geiger/firmware.elf
Retrieving maximum program size .pio/build/geiger/firmware.elf
Checking size .pio/build/geiger/firmware.elf
Advanced Memory Usage is available via "PlatformIO Home > Project Inspect"
RAM:   [==        ]  21.6% (used 70692 bytes from 327680 bytes)
Flash: [====      ]  41.8% (used 1398045 bytes from 3342336 bytes)
Building .pio/build/geiger/firmware.bin

```
